nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -894,7 +898,7 @@ define([
             busEventManager.add(runStatusListener);
             cell.execute();
             updateState('launching');
-            switchToTab('viewConfigure');
+            switchToTab('launching');
```

Comment:
switch to the launching tab on clicking the run button, as the `viewConfigure` tab can be slow to render